gobblin-iceberg/src/test/java/org/apache/gobblin/iceberg/writer/HiveMetadataWriterTest.java

```diff
@@ -80,6 +82,10 @@
 import org.apache.gobblin.stream.RecordEnvelope;
 import org.apache.gobblin.util.ClustersNames;
 import org.apache.gobblin.util.ConfigUtils;
+import org.apache.gobblin.util.function.CheckedExceptionFunction;
+import org.mockito.Mockito;
+
+import static org.mockito.Matchers.eq;
 
 
 public class HiveMetadataWriterTest extends HiveMetastoreTest {
```

Comment:
Is there a reason why we can't move this class to the hive module? Is it a dependency issue? To me it seems like one of these classes should ultimately live in the other's module but I might be overlooking something here

Reply:
The writer lives in in gobblin-hive-registration. And the test lives in gobblin iceberg.

Your suspicion is correct. The problem is that this test relies on some hive testing utilities in the iceberg module. It's a bit of a mess to move it there and the benefit is not worth it